ui/serializers_test.py

```diff
@@ -108,41 +109,75 @@ def test_collection_list_serializer():
     assert serializers.CollectionListSerializer(collection).data == expected
 
 
-@pytest.mark.parametrize('youtube', [True, False])
-@pytest.mark.parametrize('public', [True, False])
-def test_video_serializer(youtube, public):
+def get_expected_result(video):
     """
-    Test for VideoSerializer
+    Expected result for VideoSerializer
     """
-    video = factories.VideoFactory()
-    video_files = [factories.VideoFileFactory(video=video)]
-    video_thumbnails = [factories.VideoThumbnailFactory(video=video)]
-    video.is_public = public
-    if youtube and public:
-        factories.YouTubeVideoFactory(video=video)
-
-    expected = {
+    return {
         'key': video.hexkey,
         'collection_key': video.collection.hexkey,
         'collection_title': video.collection.title,
         'created_at': DateTimeField().to_representation(video.created_at),
         'multiangle': video.multiangle,
         'title': video.title,
         'description': video.description,
-        'videofile_set': serializers.VideoFileSerializer(video_files, many=True).data,
-        'videothumbnail_set': serializers.VideoThumbnailSerializer(video_thumbnails, many=True).data,
         'videosubtitle_set': [],
         'status': video.status,
         'collection_view_lists': [],
         'view_lists': [],
-        'sources': [],
+        'sources': video.sources,
         'is_private': False,
-        'is_public': public,
-        'youtube_id': (video.youtube_id if youtube and public else None)
+        'is_public': video.is_public,
+        'youtube_id': None,
+        'cloudfront_url': "",
```

Comment:
Minor nitpick, you can add these lines here:
```python
'videofile_set': serializers.VideoFileSerializer(video.videofile_set.all(), many=True).data,
'videothumbnail_set': serializers.VideoThumbnailSerializer(video.videothumbnail_set.all(), many=True).data
```

and then you don't have to set `expected['videofile_set']` or `expected['videothumbnail_set']` in the two tests that use this function.